src/main/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisParameters.java

```diff
@@ -30,6 +30,8 @@ public class OpenSecurityAnalysisParameters extends AbstractExtension<SecurityAn
 
     private String contingencyActivePowerLossDistribution = CONTINGENCY_ACTIVE_POWER_LOSS_DISTRIBUTION_DEFAULT_VALUE;
 
+    private boolean useWarmStart = USE_WARM_START_DEFAULT_VALUE;
```

Comment:
This is really a confusing naming, warm start is not something related to a SA. This is normally in a LF what we call in load flow parameters "previous value" mode